paper.md

```diff
@@ -0,0 +1,150 @@
+
+## Automated testing
+
+Automated testing is critical to ensuring the correctness of code during initial implementation and through ongoing changes. Code should be validated by a comprehensive suite of tests including unit tests, which confirm the accuracy of individual functions, and integration tests that ensure end-to-end functionality for supported use cases. A comprehensive and automated test suite is an indicator of a trustworthy software package.
+
+The LF-PPT supports automated testing by configuring several different continuous integration GitHub workflows, that run test suites automatically:
+ - at each push or pull request to ensure that changes to the current branch do not break the program’s behavior and
+ - in regularly scheduled smoke tests (usually daily) to ensure that the code has not broken due to a change of the project’s code or the behavior of its dependencies.
+
+## Documentation
+
+Thorough documentation of code allows others to use the package more readily and encourages maintainability. The LF-PPT supports the use of sphinx[^1] for integration with ReadTheDocs[^2] to automatically render Python docstrings into well formatted documentation alongside manually written documentation. Additionally, to demonstrate the use of the package in context, the LF-PPT provides automatic rendering of example Jupyter notebooks within the documentation.
+
+## Distribution
+
+Software is only useful to the broader community if other users can find, install, and update it. Package management systems such as pip[^3] or conda[^4] provide simple tools for a user to download a new software package and its dependencies from public repositories such as PyPI[^7] or conda-forge[^8]. To enable easy code installation with pip or conda the LF-PPT provides support for automatic distribution through both PyPI and conda-forge when the user applies a new git tag via the GitHub UI.
+
+## Additional code health tools
+
+The LF-PPT supports many more features than those listed here including static code analysis, performance benchmarking, code testing coverage and more. For a complete list, please see the documentation at http://lincc-ppt.readthedocs.io/. 
+
+The most notable exclusion from the LF-PPT is code. Aside from a few optional stub source and test files, the LF-PPT allows the user focus on scientific code, while supporting them with the industry best practices in maintainable software engineering.  
+
+# Usage
+
+## General usage
+
+The LINCC-Frameworks Python Project Template automates the setup of the above processes for Python projects hosted on GitHub. The only direct dependency is copier[^5] [@copier], which is used as the engine to generate new projects from the LF-PPT with a specific directory structure, the requested configuration and stub files. To hydrate a new or existing project with the LF-PPT via copier the user calls:
+
+> copier copy gh:lincc-frameworks/python-project-template <new/project/directory>
```

Comment:
Turns out, you can't suggest adding three back-ticks - kinda breaks the github UI.

Anyway. I think we should use backtick-code-blocks for this and the other command line prompts. The ">" character in markdown implies quoted text, and is rendered differently.

Reply:
Gotcha, I can do that. Thanks!